docs/project-configuration/custom-scripts.mdx

```diff
@@ -40,6 +40,12 @@ definition:
   #highlight-end
 ```
 
+:::info Getting started
+
+Follow the steps in the [Quickstart](/getting-started/quickstart) to set up a project with a supergraph, in order work with custom scripts.
```

Comment:
```suggestion
```suggestion
To work with custom scripts, follow the steps in the [Quickstart](/getting-started/quickstart) to set up a project with a supergraph.
``````